stages/org.osbuild.firewall

```diff
@@ -72,7 +76,14 @@ def main(tree, options):
     enabled_services = options.get("enabled_services", [])
     disabled_services = options.get("disabled_services", [])
 
+    default_zone = options.get("default_zone", "")
```

Comment:
The `""` is technically not needed but now a big deal.

Reply:
True, it would be `None` and the `if` statement would still work as is...